jerry-core/ecma/operations/ecma-objects.c

```diff
@@ -486,6 +558,7 @@ ecma_op_object_find_own (ecma_value_t base_value, /**< base value */
  *
  * @return ecma value if property is found
  *         ECMA_SIMPLE_VALUE_NOT_FOUND if property is not found
+ *         and do not search via the protio chain any more
```

Comment:
I don't this comment line is needed. I see what you want to say but it is obvious from the function name.